lib/Data.js

```diff
@@ -34,21 +36,17 @@ export class Suite {
     return this.tests.concat(childSuiteTests)
   }
 
-  get runtime () {
-    var status = this.status
-
-    if (status === 'skipped' || status === undefined) {
+  getRuntime () {
```

Comment:
How about a getStatus() helper function in the private scope of the module, then in the constructor, calling `this.status = getStatus(this)`?